scripts/install-calico-windows.ps1

```diff
@@ -213,6 +214,9 @@ if ($platform -EQ "ec2") {
     SetConfigParameters -OldString '$(hostname).ToLower()' -NewString "$awsNodeNameQuote"
     GetCalicoKubeConfig -SecretName 'calico-node'
 }
+if([string]::IsNullOrEmpty($platform) -and (-not [string]::IsNullOrEmpty($CalicoKubeConfigSecret))) {
+    GetCalicoKubeConfig -SecretName "$CalicoKubeConfigSecret"
```

Comment:
Can we just fix the name to `calico-node` as other platforms as there should always be a `calico-node` service account? I don't understand why a parameter is needed.

Reply:
true, that's even simpler for the user